src/sentry/integrations/tasks/vsts/subscription_check.py

```diff
@@ -0,0 +1,90 @@
+from time import time
+
+from django.core.exceptions import ObjectDoesNotExist
+
+from sentry.integrations.models.integration import Integration
+from sentry.integrations.tasks import logger
+from sentry.models.apitoken import generate_token
+from sentry.shared_integrations.exceptions import ApiError, ApiUnauthorized
+from sentry.silo.base import SiloMode
+from sentry.tasks.base import instrumented_task, retry
+
+
+@instrumented_task(
+    name="sentry.integrations.tasks.vsts.subscription_check",
+    queue="integrations.control",
+    default_retry_delay=60 * 5,
+    max_retries=5,
+    silo_mode=SiloMode.CONTROL,
+)
+@retry(exclude=(ApiError, ApiUnauthorized, Integration.DoesNotExist))
+def vsts_subscription_check(integration_id: int, organization_id: int) -> None:
```

Comment:
Same here with putting these tasks in `sentry.integrations.vsts.tasks`. We could skip renaming the task for now and do that in a followup change.